help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+| Export frequency | Streaming | Streaming destinations are “always on” API-based connections. As soon as a profile is updated in Experience Platform based on audience evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types#streaming-destinations). |
+
+{style="table-layout:auto"}
+
+## Set up Destination {#set-up-destination}
+
+>[!IMPORTANT]
+>
+>* To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage Destinations]** [access control permissions](/help/access-control/home.md#permissions).
+
+To configure a Marketo Engage instance as a destination, that instance must be in the same IMS org as your CDP instance. If you have already configured a destination, you may select an existing Marketo account to use with your new configuration. If not, click the Connector to Destination prompt, which will allow you to set the name, description, and Marketo Munchkin ID of the desired destination. Your Marketo Instance's Munchkin ID can be found in the Admin->Munchkin menu.
+
+>[!IMPORTANT]
+>
+>The user setting up the destination must have [Edit Person](https://experienceleague.adobe.com/en/docs/marketo/using/product-docs/administration/users-and-roles/descriptions-of-role-permissions#access-database) permission in the Marketo instance and partition.
+
+![Connect to Destination](../../assets/catalog/adobe/marketo-engage-person-sync/connect-to-destination.png)
+
+Once you have selected your instance, you will also need to select the Lead Partition which you want the configuration to integrate with. A [Lead Partition](https://experienceleague.adobe.com/en/docs/marketo/using/product-docs/administration/workspaces-and-person-partitions/understanding-workspaces-and-person-partitions) is a concept in Marketo Engage used to separate lead records by business concern, such as a brand or a sales region. If your Marketo subscription does not have the Workspaces and Partitions feature, or if no additional partitions have been created in your subscription, then only the Default partition will be available. A single configuration can only update lead records which exist in its configured partition.
+
+>[!IMPORTANT]
+> 
+>After a segment has been activated to the Marketo destination for the first time, backfilling profiles that already existed in the segment prior to Marketo destination activation can take [!UICONTROL up to 24 hours]. Going forward, any time profiles are added to the segment, they’ll be added to Marketo immediately.
```

Comment:
minor, but we call these audiences now instead of segments

Reply:
also, not sure why you used:
[!UICONTROL up to 24 hours]

did you want to bold or italicize this? if yes, simply use _up to 24 hours_